Assets/General/Scripts/Console/ConsoleController.cs

```diff
@@ -41,7 +41,14 @@ public class ConsoleController : MonoBehaviour
     [Header("Free look cam references")]
     [SerializeField] private GameObject _freeLookCam;
     [SerializeField] private GameObject _toggleFreeLookCamButton;
-    
+
+    [Header("Infinite Focus References")]
+    [SerializeField] private GameObject _infiniteFocusToggleButton;
+
+    //false means not focus max and true is in max focus
+    private bool _isInfocusState = true;
```

Comment:
This boolean is confusing to me. I thought this boolean was for if the player was currently focusing initially. As this is the toggle for the infinite focus perhaps the name should reflect that. Also be sure to follow the private naming convention
For example this variable would be written as "_isInFocusState" (capital F)